makefile

```diff
@@ -30,3 +30,6 @@ unit-test:
 
 gpu-unit-test:
 	bash tools/ci_testing/addons_gpu.sh
+
+tutorials-test:
```

Comment:
can you make it so this install the dependencies.
```
tutorials-test: install-ci-dependency
    docker run --rm -it -v ${PWD}:/addons -w /addons tensorflow/tensorflow:custom-op-ubuntu16 /bin/bash
```

Reply:
I am confused a bit. How does it start the tests with your commands above?

Reply:
Ah apologies I copied the wrong command in the bottom line. Please see the edited block

Reply:
Ok. Pushed the update.